.ci.yaml

```diff
@@ -3460,7 +3460,6 @@ targets:
 
   - name: Mac native_ui_tests_macos
     recipe: devicelab/devicelab_drone
-    bringup: true
```

Comment:
I would suggest we validate the changes before enabled back in prod.

Reply:
I will.  I'm making sure it passes in presubmit since `bringup:true` is skipped, and then I will change it back.